### PR TITLE
Source record data issues

### PR DESCRIPTION
**Pull request by ric2016**
Fixes for #1375, #1376, and #1377.

### CODE REVIEW

File: resources/views/edit/edit-fact.phtml [hunk 1]
```diff
@@ -15,6 +15,25 @@
...
+                    'tree' => $tree,
+                ]);
+                echo view('cards/add-shared-note', [
+                    'level' => 2,
+                    'tree' => $tree,
+                ]);
+            }
+            // SOUR:DATA facts may also take multiple EVEN.
+            if ($level1type === 'DATA') {
+                echo view('cards/add-sour-data-even', [
```
Comment (fisharebest) [inline resources/views/edit/edit-fact.phtml]: Did you forget to `git add resources/views/cards/add-sour-data-even.phtml`?

Comment (ric2016) [reply resources/views/edit/edit-fact.phtml]: Yes, sorry - It's added now.

**Comment by fisharebest** [comment]
I have merged these changes.

Thanks - and sorry it has taken me so long to find the time to review the PR.

**Comment by ric2016** [comment]
> I have merged these changes.

I'm afraid this hasn't work out correctly - E.g. `edit-fact.phtml` misses the changes for type SOUR. Other commits may also not have been merged ("branch has unmerged commits").

**Pull request closed without merging**